### PR TITLE
HACKING.adoc enhancements

### PR DESCRIPTION
**Pull request by shindere**
Typo fix and CI-related updates.

### CODE REVIEW

File: HACKING.adoc [hunk 1]
```diff
@@ -272,25 +272,34 @@ PR.
 You do not need to be an INRIA employee to open an account on this
 jenkins service; anyone can create an account there to access build
 logs, enable email notifications, and manually restart builds. If you
-would like to do this but have trouble doing it, please contact Damien
-Doligez or Gabriel Scherer.
+would like to do this but have trouble doing it, please contact
```
Comment (dra27) [inline HACKING.adoc]: Micro-nit - given that it's now an email address rather than a list of names, s/contact/email

File: HACKING.adoc [hunk 2]
```diff
@@ -272,25 +272,34 @@ PR.
...
 
-(This is the syntax to push the current `HEAD` state into the `trunk`
-reference on the specified remote.)
+4. Make your branch publicly available. For instance, you can fork
+OCaml's GitHub repository and then push "mybranch" to your fork.
+
+5. Visit https://ci.inria.fr/ocaml/job/precheck and log in. Click on
+"Build with parameters".
+
+6. Fill the REPO_URL and BRANCH fields as appropriate and run the build.
```
Comment (dra27) [inline HACKING.adoc]: Fill *in* the

**Review by gasche — APPROVED** [review]

**Review by dra27 — COMMENTED** [review]
LGTM - two very minor changes

**Comment by shindere** [comment]
@dra27 many thanks!

YOur suggestions have been integrated in commit
08cf84460ee7e7ca8b7343e6aa377129b2e17cc5

This commit also fixes the information about how to be notified of build
failures on Inria's CI.

The two commits can probably be squashed when this gets merged into
trunk. And this change is probably worth being cherry-picked on 4.06.

**Comment by gasche** [comment]
Cherry-picked in 4.06 at dc6485e8d34d5055831e151ccd588eba46068613

**Pull request merged**